src/routes/device/update-multiple-devices.ts

```diff
@@ -0,0 +1,29 @@
+import Joi from 'joi';
```

Comment:
* Shall we apply the same route naming as we use in https://github.com/withthegrid/platform-sdk/pull/287? So `updateDevices`?
* Taking https://github.com/withthegrid/platform-sdk/pull/287/files#r968069994 into account, shall we rename the route `path` to `/devices`?

Reply:
Updated